index.js

```diff
@@ -7,6 +7,7 @@ const puppeteer = require('puppeteer');
 const devices = require('puppeteer/DeviceDescriptors');
 const toughCookie = require('tough-cookie');
 
+const sleep = promisify(setTimeout);
```

Comment:
```suggestion
const delay = promisify(setTimeout);
```

Reply:
Missing delay name change in practice (use) as well. Going to commit standalone.

Reply:
Note: i decided to remove this in order to add `waitForNavigation` instead. Let me know when we can run some tests to see if it is working adequately.